src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -1273,7 +1319,7 @@ namespace winrt::TerminalApp::implementation
         // create a new window. Then, in that new window, we'll try to  set the
         // StartupActions, which will again fail, returning the correct error
         // message.
-        return WindowingBehaviorUseNew;
+        return winrt::make<FindTargetWindowResult>(WindowingBehaviorUseNew, L"");

```

Comment:
```suggestion
        return winrt::make<FindTargetWindowResult>(WindowingBehaviorUseNew, L"new");
```
Should we send the right string too? Since they mean the same thing?

Reply:
I'd rather _not_ - I don't want the recipient of the `FindTargetWindowResult` thinking that it should create a new window with the name "new".